mirage.opam

```diff
@@ -1,30 +1,24 @@
 opam-version: "1.2"
+version: "dev"
```

Comment:
afaics, the trend is to not have a `version` in opam files in repositories (unlike in distributed tarballs, where it makes sense) [same in other opam files above]